### ISSUE TITLE
GNU_SAVANNAH_SOURCE Changed

### ISSUE THREAD

**Issue by nathanhaigh** [hpcugent/easybuild-framework#1599]
Tools downloaded from `GNU_SAVANNAH_SOURCE` cause problems for me. i.e. they fail. I think it's our corporate firewall complaining of the redirections. However, since the footer of their site (e.g. http://download.savannah.gnu.org/releases/libunwind/) says:

```
Downloads will redirect to your nearest mirror site.
Files on mirrors may be subject to a replication delay of up to 24 hours.
In case of problems use http://download-mirror.savannah.gnu.org/releases/
```

I tried `http://download-mirror.savannah.gnu.org/releases/` as my `GNU_SAVANNAH_SOURCE` and things worked fine.

I'm not sure if this warrants a change in the easybuild codebase or if it's specific to my situation. Or of there's a way to specify additional URLs as a fallback?

**Comment by boegel** [comment]
I'm up for redefining `GNU_SAVANNAH_SOURCE` to use `http://download-mirror.savannah.gnu.org/releases` instea

Using this (or the current URL) as a fallback is not easy, since `GNU_SAVANNAH_SOURCE` is a single entry in `source_urls`.

@nathanhaigh: up for opening a PR for the change? See `easybuild/framework/easyconfig/templates.py`

**Comment by boegel** [comment]
fixed in #1601, thanks @nathanhaigh!
